desktop/conf.dist/hue.ini

```diff
@@ -84,6 +84,9 @@
   # Execute this script to produce the SSL password. This will be used when `ssl_password` is not set.
   ## ssl_password_script=
 
+  # Django sets X-Frame-Options HTTP header to this value. Default is 'SAMEORIGIN'.
+  ## django_x_frame_options=DENY
```

Comment:
How about doing:
## django_x_frame_options=SAMEORIGIN

and above
' Default is 'SAMEORIGIN'.' --> 'Use DENY to block all the iframes'

?

We traditionally put the default as the commented value